lab-alex-bessie-david/__test__/writer.test.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
Since this file doesn't actually do anything, it should be removed. Or...better yet, test that your writer works by writing the file, and then using fs to ensure that the written file is actually there.